src/libraries/Common/src/System/Security/Cryptography/Asn1Reader/AsnValueReader.cs

```diff
@@ -216,6 +216,11 @@ internal string ReadCharacterString(UniversalTagNumber encodingType, Asn1Tag? ex
             _span = _span.Slice(consumed);
             return ret;
         }
+
+        internal int GetRemainingLength()
```

Comment:
Since AsnReader doesn't have this, AsnValueReader shouldn't, either.  The data is achievable in other ways.

Reply:
I think I can use `AsnDecoder.ReadEncodedValue(pkcs12, AsnEncodingRules.BER, out _, out _, out int bytesConsumed)`.

Reply:
I gave the suggestion at the callsite.  `reader.PeekEncodedValue().Length`.